packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -296,13 +296,19 @@ class CreateCommand extends CreateBase {
       } else if (_getSupportedPlatformsInPlugin(projectDir).isEmpty){
         _printNoPluginMessage();
       }
-      _printAddPlatformMessage(relativePluginPath);
+
+      final List<String> _warningList = _getPlatformWarningList(requestedPlatforms);
```

Comment:
nits: ivar should be in camel case, and we can make a better name. Maybe `platformsToWarn`?

Reply:
Yes `platformToWarn` seems good